src/services/createParseConfigHost.ts

```diff
@@ -0,0 +1,18 @@
+import ts from "typescript";
+
+export const createParseConfigHost = (): ts.ParseConfigHost => {
+	return {
+		// eslint-disable-next-line @typescript-eslint/unbound-method
+		directoryExists: ts.sys.directoryExists,
+		// eslint-disable-next-line @typescript-eslint/unbound-method
+		fileExists: ts.sys.fileExists,
+		getCurrentDirectory: () => process.cwd(),
+		// eslint-disable-next-line @typescript-eslint/unbound-method
+		getDirectories: ts.sys.getDirectories,
+		// eslint-disable-next-line @typescript-eslint/unbound-method
+		readDirectory: ts.sys.readDirectory,
+		// eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
At this point I'd say just disable it in the file 😄 

Is there a tracking TS issue for correcting their typedefs? If not, up for filing one? This comes up once in a while...